lib/ingestor-api/index.ts

```diff
@@ -107,8 +107,14 @@ export class StacIngestor extends Construct {
       memorySize: 2048,
     });
 
-    props.table.grantReadWriteData(handler);
     props.dataAccessRole.grant(handler.grantPrincipal, "sts:AssumeRole");
+    handler.addToRolePolicy(
+      new iam.PolicyStatement({
+        actions: ["s3:Get*", "s3:List*"],
+        resources: ["arn:aws:s3:::*"],
+      })
+    );
```

Comment:
I don't believe that this service should need to directly connect to any S3 buckets, rather that is the job of the data access role.

Reply:
As described above, because role assumption within this handler was failing we needed to apply a broad role to support reading from public buckets until we can diagnose what is preventing us from injecting a user defined role to assume.